Dockerfile

```diff
@@ -0,0 +1,58 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+FROM postgres:9.6
+
+MAINTAINER dev@datasketches.apache.org
+
+ENV PG_MAJOR 9.6
+
+ADD . /datasketches-postgresql
+WORKDIR /datasketches-postgresql
+
+RUN echo "===> Adding prerequisites..."                      && \
+    apt-get update -y                                        && \
+    DEBIAN_FRONTEND=noninteractive                              \
+        apt-get install --no-install-recommends --allow-downgrades -y -q \
+                ca-certificates                                 \
+                build-essential git                             \
+                postgresql-server-dev-9.6                       \
+                libpq-dev=$PG_MAJOR\* libpq5=$PG_MAJOR\* libecpg-dev && \
+    echo "===> Building datasketches..."                     && \
+    git clone https://github.com/apache/incubator-datasketches-cpp.git datasketches-cpp && \
```

Comment:
I'm pretty sure Apache policy means we'll need to pin this to an official release.

Reply:
fixed